corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -306,6 +307,7 @@ hqDefine("cloudcare/js/formplayer/app", [
         var sess = WebFormSession.WebFormSession(data);
         sess.renderFormXml(data, $('#webforms'));
         $('.menu-scrollable-container').addClass("d-none");
+        $('#sidebar-and-content').addClass('remove-padding-on-mobile');
```

Comment:
Does this really have to apply only to form entry?  Wondering if we could instead remove the padding on mobile for everything.  Or if say menus do need padding, then maybe that could be part of the menu definition.